csm/core/controllers/unsupported_features.py

```diff
@@ -18,7 +18,7 @@
 from csm.core.blogic import const
 
 
-@CsmView._app_routes.view("/api/v2/unsupported_features")
+# @CsmView._app_routes.view("/api/v2/unsupported_features")
```

Comment:
If you are commenting this from routes.py then no need to comment it here. Please check once.

Reply:
Make sense 
I will uncomment this

Reply:
updated